code/modules/events/electrical_storm.dm

```diff
@@ -1,8 +1,7 @@
 /datum/round_event_control/electrical_storm
 	name = "Electrical Storm"
 	typepath = /datum/round_event/electrical_storm
-	earliest_start = 6000
-	weight = 40
+	average_time = 5 //Minor event, no permanent damage

```

Comment:
Actually it can be rather difficult I personally would prefer it if instead of all bulbs breaking in a area it was a percentage and it happens in more areas around the station. (Making half the station or so dark is hell to sort as a janitor)